src/main/resources/db/migration/V19__device_update_correlation.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE `device_update_targets`
+ADD COLUMN `correlation_id` varchar(200) NULL;
```

Comment:
Maybe it would be safer to define this as `varchar(256)` to keep it consistent with the other schemas? That's the type in the table `file_cache_requests` and in campaigner.